src/components/elements/CopyEventLink.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import copyLinkIcon from 'assets/ui/copyLinkIcon.svg'
+import copyLinkIconDark from 'assets/ui/copyLinkIconDark.svg'
+import { FcCheckmark }from 'react-icons/fc';
+
+function CopyEventLink() {
+  const url = location.href
+  const [colorTheme, setTheme] = useState(localStorage.theme);
+  const [copyConfirmed, setCopyConfirmed] = useState(false);
+
+  window.addEventListener("themeChanged", () => setTheme(localStorage.theme));
+ 
+  function handleCopyClick(){
+    navigator.clipboard.writeText(url).then(
+      () => {
+        setCopyConfirmed(true);
+        setTimeout(() => {
+          setCopyConfirmed(false);
+        }, 2000);
+      },
+      (e) => {
+        if(e) {alert('Copy error')}
+      }
+    );
+  }
+
+  return (
+    <section className="relative flex flex-row gap-x-7 justify-center items-center md:text-xl dark:text-white font-medium">
+      Copy Link
+      <button onClick={()=> {handleCopyClick()}} className="flex-none">
```

Comment:
We don't need the brackets around handle copy, just so I click=handlecopyclick with one bracket around click